game-core/src/main/java/games/strategy/engine/framework/startup/mc/ServerModel.java

```diff
@@ -389,7 +386,6 @@ private void createServerMessenger(@Nonnull final ServerConnectionProps props) {
       final ClientLoginValidator clientLoginValidator = new ClientLoginValidator(serverMessenger);
       clientLoginValidator.setGamePassword(props.getPassword());
       serverMessenger.setLoginValidator(clientLoginValidator);
-      serverMessenger.addErrorListener(this);
```

Comment:
Ditto with this removal

Reply:
Same answer, but more clearly as `IServerMessenger` is always a no-op on error listeners.